docs-site/docs/contribute-code.md

```diff
@@ -18,120 +18,70 @@ A simple way to enforce the Google coding styles is to use the linting and forma
 
 Once the extensions are installed, edit the `settings.json` file.
 
-```json 
+```json
 {
-   "[cpp]": {
-      "editor.defaultFormatter": "xaver.clang-format"
-   },
-   "cpplint.cpplintPath": "path/to/cpplint",
-
-   "editor.formatOnSave": true,
-   "python.formatting.provider": "yapf",
-   "python.linting.enabled": true,
-   "python.linting.lintOnSave": true,
-   "clang-format.language.cpp.style": "google",
-   "python.formatting.yapfArgs" : ["--style" , "{based_on_style: google}" ]
+  "[cpp]": {
```

Comment:
how many spaces should be added for indent in Docusaurus? 3 or 4?

Reply:
usually, JS files are indenting with 2 spaces
https://github.com/apache/singa-doc/blob/0f8de13fe9decc307fbb8205397edc7650e26017/docs-site/.prettierrc#L7